README.md

```diff
@@ -2,11 +2,7 @@
 Website for the BFHS-Open organization, maintained by the BFHS Computer Science Club
 
 ## Resources
-- [Jekyll Documentation](https://jekyllrb.com/docs/) - Please become familiar with how Jekyll works and how to use it <br />
-- [Basic Markdown Syntax](https://www.markdownguide.org/basic-syntax/)
-
-## TODO
-- [Decide on theme + layout](https://jekyllrb.com/docs/themes/)
-- Home page
-- About page
+- [The Odin Project Foundations](https://www.theodinproject.com/paths/foundations/courses/foundations): Basics of HTML, CSS, and beyond
+- [MDN Docs](https://developer.mozilla.org/en-US/): In-depth web development documentation
 
```

Comment:
```suggestion
- [HTML 4 Specifications](https://www.w3.org/TR/html4/): HTML Specification

```

Reply:
Not HTML5?

Reply:
didn't expect you to actually read it ;)

Reply:
Ahh yes, just blindly merging the pull request without reading the suggestions.